site_root/input/resources/examples/DocumentReference-example-research-study-document-reference-id.json

```diff
@@ -0,0 +1,88 @@
+{
```

Comment:
Could we use toy valueStrings for the rest of the resource instead of `"any-id"`, `"any-file-name"`, etc.?

Reply:
resolved - implemented.